### ISSUE TITLE
Add batch normalization option

### ISSUE THREAD

**Issue by jpsember** [jpsember/java-ml#58]

**Comment by jpsember** [comment]
Relevant:

https://androidkt.com/use-the-batchnorm-layer-in-pytorch/
